Libraries/Linking/Linking.js

```diff
@@ -57,6 +57,11 @@ class Linking extends NativeEventEmitter {
    * See https://facebook.github.io/react-native/docs/linking.html#openurl
    */
   openURL(url: string): Promise<any> {
+    if (url.toLowerCase().indexOf('https://') === 0) {
```

Comment:
nit: It'd be nice to have a comment to clarify why this needs to be done. Something like:
```JavaScript
// Android requires url protocols to be lowercase (i.e: Https:// doesn't work, but https:// does)
```